tests/core/test_cells.py

```diff
@@ -43,21 +43,11 @@
     CellType.QUAD,
     CellType.TETRA,
     CellType.VOXEL,
-    CellType.HEXAHEDRON,
 ]
-faces_types = [
-    CellType.QUAD,
-    None,
-    CellType.PIXEL,
-    None,
-    CellType.TRIANGLE,
-    CellType.PIXEL,
-    CellType.QUAD,
-]
-dims = [3, 2, 3, 2, 3, 3, 3]
-npoints = [8, 3, 8, 4, 4, 8, 8]
-nfaces = [6, 0, 6, 0, 4, 6, 6]
-nedges = [12, 3, 12, 4, 6, 12, 12, 12]
+dims = [3, 2, 3, 2, 3, 3]
+npoints = [8, 3, 8, 4, 4, 8]
+nfaces = [6, 0, 6, 0, 4, 6]
+nedges = [12, 3, 12, 4, 6, 12, 12]
 
 ids = [str(type(grid)) for grid in grids]
 cell_ids = list(map(repr, types))
```

Comment:
Isn't this unused now?
```suggestion
```

Reply:
We found two unused variables including which I found.
We must ensure the unused variable lint in `flake8` is doing its job correctly.

Reply:
I chose to use the original `cell_ids` as the numeric type is too easily confused with the run order, which is the default.

Reply:
> We must ensure the unused variable lint in flake8 is doing its job correctly.

flake8 is tricky in tests as there are many special things that are done only for tests.  For example sometimes a variable is defined only for pytest to pick up on.  It would be good to investigate this, but I suspect we will always need closer human reviews for tests.